pages/recipes/[slug].tsx

```diff
@@ -0,0 +1,92 @@
+import { useRouter } from "next/router";
```

Comment:
Unused imports and router & renderer (lines 50/51) unused as well.

Reply:
removed unused variables and imports everywhere